startall

```diff
@@ -1,3 +1,60 @@
 #!/usr/bin/env bash
 
-docker-compose -f docker-compose.yml -f keycloak.yml -f glowingbear-ssl-proxy.yml up -d
+die()
+{
+	local _ret=$2
+	test -n "$_ret" || _ret=1
+	test "$_PRINT_HELP" = yes && print_help >&2
+	echo "$1" >&2
+	exit ${_ret}
+}
+
+
+begins_with_short_option()
+{
+	local first_option all_short_options='h'
+	first_option="${1:0:1}"
+	test "$all_short_options" = "${all_short_options/$first_option/}" && return 1 || return 0
+}
+
+# THE DEFAULTS INITIALIZATION - OPTIONALS
+_arg_connector="off"
+
+
+print_help()
+{
+	printf '%s\n' "<The general help message of my script>"
```

Comment:
Perhaps `Start all Glowing Bear and TranSMART components using docker-compose`?

Reply:
:-D nice catch